app/code/Magento/Ui/view/base/web/js/lib/validation/rules.js

```diff
@@ -803,6 +803,14 @@ define([
             },
             $.mage.__('Please enter a valid date.')
         ],
+        'validate-date-range': [
+            function (value, params) {
```

Comment:
Shouldn't we make the validation logic for `validate-date-range` rule to be the same as in `lib/web/mage/validation.js` for better consistency?